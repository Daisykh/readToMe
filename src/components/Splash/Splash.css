#splash-page {
  background-image: url('../../assets/5.jpeg');
  text-align: center; 
  height: 100vh;
  position: relative;
  width: 100%;
  background-repeat: no-repeat;
  background-size: cover;
  margin-top: 0px;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  text-align: center;
}

#layer {
  background: rgba(246, 109, 95, 0.2);
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
}

#logo {
  size: 40px;
  color: #333332;
}

button {
  color: white;
  background: #66e3e1; /* Old browsers */
  background: -moz-linear-gradient(left, #66e3e1 1%, #56c5e0 100%); /* FF3.6-15 */
  background: -webkit-linear-gradient(left, #66e3e1 1%,#56c5e0 100%); /* Chrome10-25,Safari5.1-6 */
  background: linear-gradient(to right, #66e3e1 1%,#56c5e0 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
  filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#66e3e1', endColorstr='#56c5e0',GradientType=1 ); /* IE6-9 */
  box-shadow: none;
  border-style: none;
  border-radius: 5px;
  padding:0.7em 1.7em;
  margin:0px 20px 0px 20px;
  border-radius:0.2em;
}