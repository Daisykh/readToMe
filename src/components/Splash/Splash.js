import React from 'react';
import './Splash.css';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';

export const Splash = () =>  {
  return (
    <div id="splash-page">
      <div id="layer">
        <div id="blur"></div>
        <h1 id="logo">Read To Me</h1>
        <Link to="/bookshelf" className="link-button">1st Grade</Link>
        <Link to="/bookshelf" className="link-button">2nd Grade</Link>
        <Link to="/bookshelf" className="link-button">3rd Grade</Link>
      </div>
    </div>
  );
};

Splash.propTypes = {
  location: PropTypes.object
};

export default Splash;