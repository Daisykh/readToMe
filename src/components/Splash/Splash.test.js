/* eslint-disable */

import React from 'react';
import { shallow } from 'enzyme';
import Splash from './Splash';

describe('Splash', () => {
  it('should match snapshot', () => {
    const renderedComponent = shallow(<Splash />);

    expect(renderedComponent).toMatchSnapshot();
  })
})
