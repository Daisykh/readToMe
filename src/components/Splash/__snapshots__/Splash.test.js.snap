// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        id="layer"
>
        <div
                id="blur"
        />
        <h1
                id="logo"
        >
                Read To Me
        </h1>
        <Link
                className="link-button"
                replace={false}
                to="/bookshelf"
        >
                1st Grade
        </Link>
        <Link
                className="link-button"
                replace={false}
                to="/bookshelf"
        >
                2nd Grade
        </Link>
        <Link
                className="link-button"
                replace={false}
                to="/bookshelf"
        >
                3rd Grade
        </Link>
</div>,
      "id": "splash-page",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            id="blur"
/>,
          <h1
            id="logo"
>
            Read To Me
</h1>,
          <Link
            className="link-button"
            replace={false}
            to="/bookshelf"
>
            1st Grade
</Link>,
          <Link
            className="link-button"
            replace={false}
            to="/bookshelf"
>
            2nd Grade
</Link>,
          <Link
            className="link-button"
            replace={false}
            to="/bookshelf"
>
            3rd Grade
</Link>,
        ],
        "id": "layer",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "id": "blur",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Read To Me",
            "id": "logo",
          },
          "ref": null,
          "rendered": "Read To Me",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "1st Grade",
            "className": "link-button",
            "replace": false,
            "to": "/bookshelf",
          },
          "ref": null,
          "rendered": "1st Grade",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "2nd Grade",
            "className": "link-button",
            "replace": false,
            "to": "/bookshelf",
          },
          "ref": null,
          "rendered": "2nd Grade",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "3rd Grade",
            "className": "link-button",
            "replace": false,
            "to": "/bookshelf",
          },
          "ref": null,
          "rendered": "3rd Grade",
          "type": [Function],
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          id="layer"
>
          <div
                    id="blur"
          />
          <h1
                    id="logo"
          >
                    Read To Me
          </h1>
          <Link
                    className="link-button"
                    replace={false}
                    to="/bookshelf"
          >
                    1st Grade
          </Link>
          <Link
                    className="link-button"
                    replace={false}
                    to="/bookshelf"
          >
                    2nd Grade
          </Link>
          <Link
                    className="link-button"
                    replace={false}
                    to="/bookshelf"
          >
                    3rd Grade
          </Link>
</div>,
        "id": "splash-page",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              id="blur"
/>,
            <h1
              id="logo"
>
              Read To Me
</h1>,
            <Link
              className="link-button"
              replace={false}
              to="/bookshelf"
>
              1st Grade
</Link>,
            <Link
              className="link-button"
              replace={false}
              to="/bookshelf"
>
              2nd Grade
</Link>,
            <Link
              className="link-button"
              replace={false}
              to="/bookshelf"
>
              3rd Grade
</Link>,
          ],
          "id": "layer",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "id": "blur",
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Read To Me",
              "id": "logo",
            },
            "ref": null,
            "rendered": "Read To Me",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "1st Grade",
              "className": "link-button",
              "replace": false,
              "to": "/bookshelf",
            },
            "ref": null,
            "rendered": "1st Grade",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "2nd Grade",
              "className": "link-button",
              "replace": false,
              "to": "/bookshelf",
            },
            "ref": null,
            "rendered": "2nd Grade",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "3rd Grade",
              "className": "link-button",
              "replace": false,
              "to": "/bookshelf",
            },
            "ref": null,
            "rendered": "3rd Grade",
            "type": [Function],
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
